mysite/urls.py

```diff
@@ -0,0 +1,22 @@
+"""mysite URL Configuration.
```

Comment:
Don't keep this whole boilerplate here. Use an actual description.

Reply:
What should I write?

Reply:
Something that explains what the module is for, not django usage instructions.